NepovinnyUkolOOP/Obdelnik.cs

```diff
@@ -18,7 +18,7 @@ private set
             {
                 if (value == 0)
                 {
-                    Console.WriteLine("Sirka nesmí být nula. Hodnota byla nastavena na 1.");
```

Comment:
Pokud chceme, aby se sirka uz po inicializaci nemohla nastavit, tak se na to pouziva klicove slovo `init`. Private set porad umoznuje nastavit hodnotu z teto tridy `Obdelnik` :) To samy pro vysku
```csharp
 public double Sirka
    {
        get => sirka;
        init
        {
            if (value == 0)
            {
                Console.WriteLine("Sirka nesmi byt nula. Hodnota byla nastavena na 1.");
                sirka = 1;
            }
            else
            {
                sirka = value;
            }
        }
    }
```